### ISSUE TITLE
Link to Drive document under "CryoET Data Portal Data and Schema" is dead

### ISSUE THREAD

**Issue by jpellman** [chanzuckerberg/cryoet-data-portal#401]
Within the portal documentation [here](https://chanzuckerberg.github.io/cryoet-data-portal/index.html#cryoet-data-portal-data-and-schema), the link to the schema/portal description is for a Google Drive document that does not exist (https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing).  Perhaps this document is not shared with the public if it does, in fact, exist?

**Comment by junxini** [comment]
@jpellman thank you for posting about this bug. We recently had an org-wide document migration causing the error that you're experiencing. Our team is taking this opportunity to make sure the metadata schema documentation is up to date with the latest version before sharing it back out. We'll update this Github issue with the latest status. cc @dgmccart

**Comment by junxini** [comment]
once @dgmccart has the latest document updated, @kne42 will you also re-link on the FAQ page (2nd to last section) as well?

**Comment by kne42** [comment]
marking this as blocked as dannielle is still making changes to the document before we can publish

**Comment by dgmccart** [comment]
@manasaV3 and @uermel are taking this on. For now, we could link to the [API documentation](https://chanzuckerberg.github.io/cryoet-data-portal/python-api.html).

**Comment by junxini** [comment]
> once [@dgmccart](https://github.com/dgmccart) has the latest document updated, [@kne42](https://github.com/kne42) will you also re-link on the FAQ page (2nd to last section) as well?

@kne42 could you help with the above by updating the link to the API documentation as the temporary location?

**Comment by junxini** [comment]
@kne42 i've opened up a new ticket for re-linking with metadata schema once it's ready (#507). feel free to close this bug out once you've added the temporary API documentation to both links
